packages/terra-alert/src/Alert.module.scss

```diff
@@ -68,7 +69,6 @@
 
   .title {
     display: var(--terra-alert-title-display);
-    float: var(--terra-alert-title-float, left);
```

Comment:
Not a fan of removing this or adjusting the component, only need to update the example contents.

Reply:
What was the intention behind this? If you are using flexbox, floats don't act on the box model.

Reply:
It wasn't so much due to browsers not being able to support flex, but more due to uncontrollable nature of the possible children content that can be provided. 

Traditional float was used instead of flex so the title can be inserted before the message text (or any children content) and essentially function as if is was set as `inline` and delivered with the children/message. It allows for normal block content to be inserted as the alert content, which since it is pure children, is largely unable to be controlled or understood.